src/Helpers/OrderBuilderTrait.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace GingerPluginSdk\Helpers;
+
+use GingerPluginSdk\Entities\Order;
+use GingerPluginSdk\Response\GingerHTTPResponse;
+use GingerPluginSdk\Response\GingerHTTPResponseBody;
+use JetBrains\PhpStorm\ArrayShape;
+
+trait OrderBuilderTrait
+{
+    protected Order $order;
+
+    public function setOrder(Order $order): static
+    {
+        $this->order = $order;
+        return $this;
+    }
+
+    /**
+     * @throws \Exception
+     */
+    public function debugOrder()
```

Comment:
This looks like a strange feature for a library. Could this be done without the debug method?

Reply:
Ofk can, it's just boring to me write print_r(bla-bla-bla), with this method i just need to put two buttons and that's it. I know that's lazy, think it's will be removed at the end,